src/main/java/com/hubspot/smtp/client/SmtpSession.java

```diff
@@ -149,74 +153,94 @@ public boolean isEncrypted() {
   }
 
   public CompletableFuture<SmtpClientResponse[]> send(String from, String to, MessageContent content) {
+    return send(from, Collections.singleton(to), content);
+  }
+
+  public CompletableFuture<SmtpClientResponse[]> send(String from, Collection<String> recipients, MessageContent content) {
     Preconditions.checkNotNull(from);
-    Preconditions.checkNotNull(to);
+    Preconditions.checkNotNull(recipients);
+    Preconditions.checkArgument(recipients.size() > 0, "recipients must be > 0");
```

Comment:
nit: typically we ask `!collection.isEmpty()` rather than looking at the size itself - it's what you're trying to assert here.